### PR TITLE
Add: SAGAUserGroupAssociation.SAGAGIS version 7.9.1.0

### PR DESCRIPTION
**Pull request by quhxl**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/20527)

### CODE REVIEW

File: manifests/s/SAGAUserGroupAssociation/SAGAGIS/7.9.1.0/SAGAUserGroupAssociation.SAGAGIS.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+﻿# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+
+PackageIdentifier: SAGAUserGroupAssociation.SAGAGIS
+PackageVersion: 7.9.1.0
```
Comment (ItzLevvie) [inline manifests/s/SAGAUserGroupAssociation/SAGAGIS/7.9.1.0/SAGAUserGroupAssociation.SAGAGIS.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 7.9.1
```

File: manifests/s/SAGAUserGroupAssociation/SAGAGIS/7.9.1.0/SAGAUserGroupAssociation.SAGAGIS.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+﻿# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: SAGAUserGroupAssociation.SAGAGIS
+PackageVersion: 7.9.1.0
```
Comment (ItzLevvie) [inline manifests/s/SAGAUserGroupAssociation/SAGAGIS/7.9.1.0/SAGAUserGroupAssociation.SAGAGIS.installer.yaml:5]:
```suggestion
PackageVersion: 7.9.1
```

File: manifests/s/SAGAUserGroupAssociation/SAGAGIS/7.9.1.0/SAGAUserGroupAssociation.SAGAGIS.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+﻿# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: SAGAUserGroupAssociation.SAGAGIS
+PackageVersion: 7.9.1.0
```
Comment (ItzLevvie) [inline manifests/s/SAGAUserGroupAssociation/SAGAGIS/7.9.1.0/SAGAUserGroupAssociation.SAGAGIS.yaml:5]:
```suggestion
PackageVersion: 7.9.1
```

File: manifests/s/SAGAUserGroupAssociation/SAGAGIS/7.9.1.0/SAGAUserGroupAssociation.SAGAGIS.installer.yaml (hunk 2 above)
Comment (ItzLevvie) [reply manifests/s/SAGAUserGroupAssociation/SAGAGIS/7.9.1.0/SAGAUserGroupAssociation.SAGAGIS.installer.yaml:5]: image

File: manifests/s/SAGAUserGroupAssociation/SAGAGIS/7.9.1.0/SAGAUserGroupAssociation.SAGAGIS.locale.en-US.yaml (hunk 1 above)
Comment (ItzLevvie) [reply manifests/s/SAGAUserGroupAssociation/SAGAGIS/7.9.1.0/SAGAUserGroupAssociation.SAGAGIS.locale.en-US.yaml:5]: image

File: manifests/s/SAGAUserGroupAssociation/SAGAGIS/7.9.1.0/SAGAUserGroupAssociation.SAGAGIS.yaml (hunk 3 above)
Comment (ItzLevvie) [reply manifests/s/SAGAUserGroupAssociation/SAGAGIS/7.9.1.0/SAGAUserGroupAssociation.SAGAGIS.yaml:5]: image

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by quhxl** [comment]
Why is this failing? It passed and I just changed the version number...

**Pull request closed without merging**